tsconfig.json

```
@@ -21,5 +21,5 @@
     "esModuleInterop": true,                   /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
     "emitDeclarationOnly": true
   },
-  "files": ["./src/index.ts"]
+  "files": ["./src/index.ts", "./src/relay.ts"]
```

```suggestion
  "files": ["./src"]
```

This would cover all future files? Any reason for narrowing it down?

---

Some times you don't actually want to expose helper files/implementation details because if people start to rely on them it could break backward compat